BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/Model/PartsPickerAdapter.java

```diff
@@ -0,0 +1,140 @@
+package com.blueprint.foe.beetracker.Model;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
+import android.graphics.drawable.LayerDrawable;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawable;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawableFactory;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import com.blueprint.foe.beetracker.OnBeePartSelectedListener;
+import com.blueprint.foe.beetracker.R;
+import com.blueprint.foe.beetracker.SubmissionActivity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A RecyclerView Adapter to hold bee part images. Used by LearnActivity.
+ * Looks like a horizontal ListView with round bee part images. The images optionally have a
+ * green circle around them to indicate they are selected. In order to render them according to
+ * the design, a grey background is also generated to go behind the transparent bee resource.
+ */
+public class PartsPickerAdapter extends RecyclerView.Adapter<PartsPickerAdapter.ViewHolder> {
+    private static String TAG = PartsPickerAdapter.class.toString();
+    private static int SIZE = 80;
+    private BeePartType type;
+    private List<BeePart> beeParts;
+    private OnBeePartSelectedListener mListener;
+
+    public enum BeePartType {
```

Comment:
I feel this should be outside of PartsPickerAdapter since BeePart seems generic enough